File: src/NetGet.java
import javax.net.ssl.HttpsURLConnection;
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.net.URL;

public class NetGet {
    public static String getOneLine(String webAdress) {
        URL url;
        HttpsURLConnection conn = null;
        BufferedReader rd;
        String line;
        StringBuilder result = new StringBuilder();
        try {
            url = new URL(webAdress);
            conn = (HttpsURLConnection) url.openConnection();
            conn.setRequestMethod("GET");
            int responseCode = conn.getResponseCode();
            System.out.println("Код ответа сервера (RESPONSE CODE) = " + responseCode);
            rd = new BufferedReader(new InputStreamReader(conn.getInputStream()));
            while ((line = rd.readLine()) != null) {
                result.append(line);
            }
            rd.close();
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            try {
                conn.disconnect();
            } catch (NullPointerException e) {
                e.printStackTrace();
            }
        }
        return result.toString();
    }
}

File: src/NetGet.kt
import javax.net.ssl.HttpsURLConnection
import java.io.BufferedReader
import java.io.InputStreamReader
import java.net.URL

    fun getOneLine(webAdress: String): String {
        val url: URL
        var conn: HttpsURLConnection? = null
        val rd: BufferedReader
        var line: String
        val result = StringBuilder()
        try {
            url = URL(webAdress)
            conn = url.openConnection() as HttpsURLConnection
            conn.requestMethod = "GET"
            val responseCode = conn.responseCode
            println("Код ответа сервера (RESPONSE CODE) = $responseCode")
            rd = BufferedReader(InputStreamReader(conn.inputStream))
            line = rd.readLine()
            while (line != null) {
                result.append(line)
                line = rd.readLine()
            }
            rd.close()
        } catch (e: Exception) {
            e.printStackTrace()
        } finally {
            try {
                conn!!.disconnect()
            } catch (e: NullPointerException) {
                e.printStackTrace()
            }

        }
        return result.toString()
    }
